src/steps/sse.ts

```diff
@@ -81,18 +86,16 @@ export default async function (
     })
 
     const messages: MessageEvent[] = []
+    const expectedMessages: Set<string> | undefined = params.check?.messages
+      ? new Set(params.check?.messages?.map((m) => m.id))
+      : undefined
 
-    const timeout = setTimeout(() => {
+    // Closes the `EventSource` and exits as "passed"
+    function end () {
```

Comment:
use arrow functions for inline functions

Reply:
I changed it to `const end = () => {`, is it what you meant?